dev/diffs/3.4.2.diff

```diff
@@ -352,7 +352,7 @@ index 7dec558f8df..840dda15033 100644
      assert(exchanges.size == 2)
    }
 diff --git a/sql/core/src/test/scala/org/apache/spark/sql/DynamicPartitionPruningSuite.scala b/sql/core/src/test/scala/org/apache/spark/sql/DynamicPartitionPruningSuite.scala
-index f33432ddb6f..060f874ea72 100644
+index f33432ddb6f..9cf7a9dd4e3 100644
```

Comment:
I guess we need the same change for `3.4.3.diff` ?

Reply:
Yes